app/services/induction/enrol.rb

```diff
@@ -41,4 +47,10 @@ def preferred_identity
   def schedule_start_date
     participant_profile.schedule.milestones.first.start_date
   end
+
+  def reactivate_profile!
+    ParticipantProfileState.create!(participant_profile: participant_profile,
```

Comment:
If we remove the validations, I might consider renaming this method because we are technically not reactivating anything - we are creating a new induction record and record the new participant_profile_state - will consider this is another PR